src/navigator/NavigationService.ts

```diff
@@ -224,7 +226,7 @@ export async function isBottomSheetVisible(screen: Screens) {
 }
 
 interface NavigateHomeOptions {
-  params?: StackParamList[Screens.DrawerNavigator]
+  params?: StackParamList[Screens.DrawerNavigator] | StackParamList[Screens.TabNavigator]
```

Comment:
This type is kind of a problem, as it masks a potential issue. The `initialScreenName` accepted by the two screens are actually going to be different. What if a caller passes an `initialScreenName` that only applies to the DrawerNavigator. E.g., we have a the CYA screen passing `DAppsExplorerScreen` and `ExchangeHomeScreen`, would it be on the caller to also get the gate and pass the appropriate screen name?

Reply:
It looks like react-navigation handles internally. I've tested navigating from the CYA screen to each option and the correct screen is served for both paths for the `use_tab_navigator` feature gate.

Reply:
guess this might be broken now with changing the screen names, but we can fix it in ACT-1110